ncm-spma: yumdnf backend: fails while trying to expand groups

* Quattor release: 21.4.0
* OS: CentOS Stream release 8
* SPMA_BACKEND set to 'yumdnf' 
* Error message found in component-spma.log:
`[ERROR] Failed Group expansion: usage: dnf repoquery [-c [config file]] [-q] [-v] [--version]
...
dnf repoquery: error: unrecognized arguments: -g --grouppkgs`

--- a/ncm-spma/src/main/perl/spma/yumdnf.pm
+++ b/ncm-spma/src/main/perl/spma/yumdnf.pm
@@ -10,6 +10,8 @@ use constant YUM_CONF_FILE => "/etc/dnf/dnf.conf";
 
 use constant DNF_MODULES_DIR => "/etc/dnf/modules.d";
 
+use constant REPOGROUP => qw(true);
+
 use constant REPOQUERY_FORMAT => qw(--nevra);
 
 use constant REPO_DEPS => qw(repoquery -C --requires --resolve --qf %{NAME};%{ARCH});

